### PR TITLE
CMM 5 cards

### PR DESCRIPTION
**Pull request by Simisays**
I can't seem to figure out Demon of Fate's Design "Once during each of your turns,"

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/boon_of_the_spirit_realm.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Boon of the Spirit Realm
+ManaCost:3 W W
+Types:Enchantment
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self,Enchantment.Other+YouCtrl | Execute$ TrigCounter | TriggerDescription$ Constellation — Whenever CARDNAME or another enchantment enters the battlefield under your control, put a blessing counter on CARDNAME.
+SVar:TrigCounter:DB$ PutCounter | CounterType$ BLESS | CounterNum$ 1
+S:Mode$ Continuous | Affected$ Creature.YouCtrl | AddPower$ X | AddToughness$ X | Description$ Creatures you control get +1/+1 for each blessing counter on CARDNAME
+SVar:X:Count$CardCounters.BLESS
+DeckHints:Type$Enchantment
+DeckHas:Ability$Counters
+Oracle:Constellation — Whenever Boon of the Spirit Realm or another enchantment enters the battlefield under your control, put a blessing counter on Boon of the Spirit Realm./nCreatures you control get +1/+1 for each blessing counter on Boon of the Spirit Realm.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/boon_of_the_spirit_realm.txt:10]: /n

File: forge-gui/res/cardsfolder/upcoming/boon_of_the_spirit_realm.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Name:Boon of the Spirit Realm
+ManaCost:3 W W
+Types:Enchantment
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self,Enchantment.Other+YouCtrl | Execute$ TrigCounter | TriggerDescription$ Constellation — Whenever CARDNAME or another enchantment enters the battlefield under your control, put a blessing counter on CARDNAME.
+SVar:TrigCounter:DB$ PutCounter | CounterType$ BLESS | CounterNum$ 1
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/boon_of_the_spirit_realm.txt:5]: invalid type

File: forge-gui/res/cardsfolder/upcoming/demon_of_fates_design.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Name:Demon of Fate's Design
+ManaCost:4 B B
+Types:Enchantment Creature Demon
+PT:6/6
+K:Flying
+K:Trample
+S:Mode$ Continuous | Affected$ Card.Enchantment+YouCtrl | MayPlayLimit$ 1 | AddKeyword$ Alternative Cost:PayLife<ConvertedManaCost> | AffectedZone$ Hand,Graveyard,Exile,Library,Command | Description$ Once during each of your turns, you may cast an enchantment spell by paying life equal to its mana value rather than paying its mana cost.
```
[the remark this answers is not in the document]
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/demon_of_fates_design.txt:7]: hmm don't really know how to implement the pay life part. I know PlayCost is a param for play ....

[the remark this answers is not in the document]
Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/demon_of_fates_design.txt:7]: thanks

File: forge-gui/res/cardsfolder/upcoming/demon_of_fates_design.txt [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Name:Demon of Fate's Design
+ManaCost:4 B B
+Types:Enchantment Creature Demon
+PT:6/6
+K:Flying
+K:Trample
+S:Mode$ Continuous | Affected$ Card.Enchantment+YouCtrl | MayPlayLimit$ 1 | MayPlay$ True | MayPlayAltManaCost$ PayLife<ConvertedManaCost> | AffectedZone$ Hand,Graveyard,Exile,Library,Command | Description$ Once during each of your turns, you may cast an enchantment spell by paying life equal to its mana value rather than paying its mana cost.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/demon_of_fates_design.txt:7]: `MayPlayDontGrantZonePermissions$ True`

**Review by tool4ever — APPROVED** [review]

**Pull request merged**